src/input/code_templates/c/static_code/exi_bitstream_h.ctc

```diff
@@ -60,6 +61,10 @@ typedef struct {
     uint8_t bit_count;
     size_t byte_pos;
 
+    /* flags for reset and length function */
+    uint8_t _init_called;
+    size_t _flag_byte_pos;
```

Comment:
are these fields ever set anywhere?

Reply:
You are right, actually the values should be set in the init function. Somehow they didn't end up there. I will fix that.